lib/datasource/index.ts

```diff
@@ -131,10 +256,11 @@ export async function getDigest(
   config: DigestConfig,
   value?: string
 ): Promise<string | null> {
+  const datasource = await load(config.datasource);
   const lookupName = config.lookupName || config.depName;
-  const { registryUrls } = config;
-  return (await load(config.datasource)).getDigest(
-    { lookupName, registryUrls },
+  const registryUrls = resolveRegistryUrls(datasource, config.registryUrls);
+  return datasource.getDigest(
+    { lookupName, registryUrl: registryUrls[0] },
```

Comment:
Is this always the right url?

Reply:
For now it is.. Docker is the only datasource using registryUrls and it uses the first. Best solution would be to save the registryUrl value during getReleases

Reply:
Sounds good, we should open a ticket for this to remember